types/store/lhs.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+export type LhsViewState = {
+    isOpen: boolean;
+
+    // Static items (e.g. Threads, Insights, etc.)
+    currentStaticItemId: string;
+    staticItems: LhsStaticItem[];
```

Comment:
Somewhat repeating my previous comment, but I don't think `staticItems` need to live in the state